src/framework/mlt_producer.h

```diff
@@ -80,6 +80,25 @@ struct mlt_producer_s
 	 */
 	int ( *get_frame )( mlt_producer, mlt_frame_ptr, int );
 
+	/** Seek to s specified position (virtual function).
+	 *
+	 * \param mlt_producer a producer
+	 * \param mlt_position a frame pointer by reference
```

Comment:
a `mlt_position` is not a "frame pointer by reference"

Reply:
done